install.sh

```diff
@@ -101,7 +103,7 @@ fi
 # Fix emonhub log file permissions
 if [ -d /var/log/emonhub ]; then
```

Comment:
If the folder doesn't exist - no action - this is wrong isn't it?

Reply:
Can you open a new issue for this?